core/mule.c

```diff
@@ -58,6 +58,10 @@ void uwsgi_mule(int id) {
 		uwsgi.mules[id - 1].pid = getpid();
 		uwsgi.mypid = uwsgi.mules[id - 1].pid;
 
+		if (!uwsgi.mule_msg_recv_size)
+			uwsgi.mule_msg_recv_size = 65536;
```

Comment:
Would it make sense to place this assignment in `uwsgi_init_default`? That way the mule won't need to check whether the value is configured here and worker processes will have the default value (instead of 0) which is nice when querying the value from plugins etc.